DIMS/GenerateExcel.R

```diff
@@ -125,7 +125,9 @@ outlist <- rbind(combi_pos_neg, tmp_pos_left, tmp_neg_left)
 outlist <- outlist %>% arrange(rownames(outlist))
 
 # Filter for biological relevance
-rlvnc_in_list <- rlvnc %>% filter(HMDB_key %in% rownames(outlist))
+peaks_in_list <- which(rownames(outlist) %in% rlvnc$HMDB_key)
+rlvnc_in_list <- rlvnc %>% filter(HMDB_key %in% rownames(outlist)[peaks_in_list])
+rlvnc_in_list <- rlvnc_in_list %>% rename(sec_HMBD_ID_rlvnc = sec_HMDB_ID)
```

Comment:
```suggestion
rlvnc_in_list <- rlvnc %>%
    filter(HMDB_key %in% rownames(outlist)) %>%
    rename(sec_HMBD_ID_rlvnc = sec_HMDB_ID)
```

Unless I missed something, this would be a bit cleaner :)

Reply:
Yes, this would be cleaner. However, the variable peaks_in_list is also needed further along in the script. Lines 129 and 130 could indeed be combined to one line of code.